vizro-core/tests/unit/vizro/actions/conftest.py

```diff
@@ -103,3 +103,32 @@ def managers_one_page_one_graph_with_dict_param_input(scatter_matrix_chart):
         ],
     )
     Vizro._pre_build()
+
```

Comment:
If this is only used in the `collapse_expand_containers` test then let's move it to that file rather than putting it in conftest.